app/controllers/calls_controller.rb

```diff
@@ -20,17 +20,17 @@ def start
   # CallSid - default param from twilio (required)
   def new_connection
     response = Twilio::TwiML::Response.new do |r|
-      if active_call.next_target.present?
+      if active_call.next_target.present? && !active_call.exceeded_max_connections?
         connection = active_call.create_connection!
-        # r.Play AudioFileFetcher.file_for_key('connecting_local')
-        r.Play AudioFileFetcher.file_for_key('connecting_to_representative')
-        r.Play AudioFileFetcher.file_for_key('star_to_disconnect')
+        # r.Play AudioFileFetcher.audio_url_for_key('connecting_local')
+        r.Play AudioFileFetcher.audio_url_for_key('connecting_to_representative')
+        r.Play AudioFileFetcher.audio_url_for_key('star_to_disconnect')
         target_number = ENV['FAKE_CONGRESS_NUMBER'] || connection.legislator.phone
         r.Dial target_number, 'action' => calls_connection_gather_prompt_url, 'hangupOnStar' => true
       else
-        audio_key = active_call.connections.size >= 5 ? 'closing_message' : 'no_targets'
-        r.Play AudioFileFetcher.file_for_key(audio_key)
-        r.Play AudioFileFetcher.file_for_key('goodbye')
+        audio_key = active_call.exceeded_max_connections? ? 'closing_message' : 'no_targets'
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.